java/src/com/ibm/streamsx/topology/generator/spl/OperatorGenerator.java

```diff
@@ -89,6 +90,25 @@ private static void portTypesNote(JSONObject op, StringBuilder sb) {
         }
     }
 
+    private void viewAnnotation(JSONObject op, StringBuilder sb){
+	JSONObject config = (JSONObject)op.get("config");
+	if(config == null)
+	    return;
+	JSONObject viewConfig = (JSONObject)config.get("viewConfig");
```

Comment:
This layout is limiting to a single view on an operator, can multiple views exist on an operator, e.g. a view per port?